eo-maven-plugin/src/main/java/org/eolang/maven/name/OnDefault.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven.name;
+
+import org.eolang.maven.hash.CommitHash;
+
+/**
+ * Default object name that just split given raw string to name and hash.
+ *
+ * @since 0.31.0
+ */
+public final class OnDefault implements ObjectName {
```

Comment:
@maxonfjvipon I believe this change should be done in a separate PR. Please take a look that issue:
https://github.com/objectionary/eo/issues/2420

Reply:
@volodya-lombrozo I'm not sure in such particular case. `OnDefault` is just one more `ObjectName` without complex behaviour.

Reply:
@maxonfjvipon From the issue description:
> Apparently we have to create a class which will parse raw string into two parts value and 
optional version.

We have the same `split` function among many classes (The same we can say about `DELIMETER` static field). Hence, I believe this issue exactly about this.

Reply:
@volodya-lombrozo I suggest to extend the #2420 so it affected this class too

Reply:
@volodya-lombrozo I believe adding `OnDefault` class is not about solving the problem #2420. It's just one more object name. Yes, with code duplication, that will be somehow resolved after solving #2420.

Reply:
@maxonfjvipon Let's do it for now. But I would recommend to solve #2420 issue first and then return to that PR.

Reply:
@volodya-lombrozo yes, let's leave this as it. Object versioning has higher priority than refactoring at the moment